src/test/java/seedu/parser/ParserTest.java

```diff
@@ -36,8 +36,26 @@ void splitCommandTerm_validCommand_success() throws IncompleteCommandException {
                 "add n/`ITEM_NAME` s/`SERIAL_NUMBER` t/`TYPE` c/`COST` pf/`PURCHASED_FROM` "
                         + "pd/`PURCHASED_DATE`");
         assertEquals(expectedResult, actualResult);
-        assertEquals(expectedResult.get(0), actualResult.get(0));
-        assertEquals(expectedResult.get(1), actualResult.get(1));
+    }
+
+    @Test
+    void splitCommandTerm_listCommandOnly_success() throws IncompleteCommandException {
+        ArrayList<String> expectedResult = new ArrayList<>(
+                Arrays.asList("list", null)
+        );
+        ArrayList<String> actualResult = parser.splitCommandTerm(
+                "list");
+        assertEquals(expectedResult, actualResult);
+    }
+
+    @Test
+    void splitCommandTerm_listCommandExtraWordsRetained_success() throws IncompleteCommandException {
```

Comment:
Is this test for the list t/`type`? if it is, do we still need it?

Reply:
It was originally for that, but I've modified and retained it here just as an additional test. It's expected that a command <code>list t/\`SPEAKER\`</code> will return the ArrayList of <code>"list"</code> and <code>t/\`SPEAKER\`</code>. This is to be processed by `parseCommand`.